InnerEye/ML/deep_learning_config.py

```diff
@@ -435,6 +434,11 @@ class TrainerParams(CudaAwareConfig):
                                                            doc="Save epoch checkpoints when epoch number is a multiple "
                                                                "of recovery_checkpoint_save_interval. The intended use "
                                                                "is to allow restore training from failed runs.")
+    save_last_k_recovery_checkpoints: int = param.Integer(default=1, bounds=(-1, None),
```

Comment:
nitpicking: Maybe rename to recovery_checkpoints_save_last_k, so that all params to do with recovery have a shared prefix?

Reply:
yup